front/gatsby/tsconfig.json

```diff
@@ -3,8 +3,9 @@
     "baseUrl": "../..",
     "paths": {
       "@substrate/context": ["front/context/src"]
-    }
+    },
+    "typeRoots": ["./design-system.d.ts"]
```

Comment:
Could you try without this line? Somehow deleting it still make tsc work on my machine